samples/client/petstore/cpp-restsdk/CMakeLists.txt

```diff
@@ -4,16 +4,29 @@ project(cpprest-petstore)
 set(CMAKE_VERBOSE_MAKEFILE ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
+if (UNIX)
 
-set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -fPIC -Wall -Wno-unused-variable")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -std=c++14 -Wall -Wno-unused-variable")
+message(STATUS "Building for Linux/Unix")
 
+set(CMAKE_BUILD_TYPE Debug)
```

Comment:
This is generally a bad idea. I'd leave build type out, user has to specify it IMO. Unless we have a good reason to do so.

Reply:
This is for the test app, and not the client so I would leave it as is.